dev/breeze/src/airflow_breeze/utils/selective_checks.py

```diff
@@ -1039,3 +1040,9 @@ def providers_compatibility_checks(self) -> str:
                 if check["python-version"] in self.python_versions
             ]
         )
+
+    @cached_property
+    def is_committer_build(self):
+        if NON_COMMITTER_BUILD_LABEL in self._pr_labels:
+            return False
+        return self._github_actor in COMMITTERS
```

Comment:
Is there any protection to prevent the no-committer user from changing this value to True?

Reply:
No there is no way, but.... aI also realized that this change is wrong because the "selective checks" are run AFTER teh scripts are restored... Let me change it and explain :)

Reply:
Ah yeah... That's quite a bit more complex than that ... I will convert that one to draft (this was not a security risk, but it would not work either).

Reply:
OK. I applied a number of cleanups and clairifcations and comments in the workflow describing what's going on. I also found out along the way that the part where DEFAULT_BRANCH and DEFAULT_CONSTRAINTS_BRANCH retrieval can be simplified, after some past changes in the way how "pull request workflow" uses now target_branhc by default and not "main" as it used to be, so we do not have to use evn variables to pass it now.

Reply:
And yes. The answer is `yes` :D

Reply:
Great!